vacasa/connect/connect.py

```diff
@@ -1078,7 +1078,7 @@ def create_contract(self,
             owners: List of objects that contain three properties 'percentage_ownership', 'tax_ownership', 'contact_id'
             created_by: ID of logged in user
             start_date: Start date of the contract
-            end_date: End date of a contract (By default will be '01/01/2099'
+            end_date: End date of a contract (By default will be '2099-1-1'
```

Comment:
Just to avoid any confusion.
```suggestion
            end_date: End date of a contract (By default will be '2099-01-01'
```